src/commands/basic/prev.js

```diff
@@ -17,6 +20,7 @@ module.exports = {
 		await interaction.deferReply();
 		queue.setMetadata(interaction);
 
+		handleNext();
```

Comment:
our commands dont need to know about inner list's state, they also must not manage it in any way. changing of a list is a responsibility of TrackList class. Trigger to change is /list command